config/prod/mep-lincs-s3.yaml

```diff
@@ -3,8 +3,15 @@
 
 # For use in the MEP-LINCS Synapse Project https://www.synapse.org/mep_lincs
 
-template_path: "remote/s3-bucket.yaml"
+template:
+  type: "http"
+  url: "https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.2.7/s3-bucket.yaml"
 stack_name: "mep-lincs-s3"
+stack_tags:
+  Department: "SysBio"
+  Project: "MEP-LINCS"
+  OwnerEmail: "larsson.omberg@sagebase.org"
+  CostCenter: "Other / 000001"
```

Comment:
Where's the "Other" cost center?

Reply:
it's here https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/tags/SageFinancialProgramCodes.json#L19

but this is another guess,  @larssono would you be able to suggest a cost center tag or this? options are here: https://github.com/Sage-Bionetworks/aws-infra/tree/master/templates/tags

Reply:
@zaro0508 I'm not asking "where is the list of program codes."  I'm asking, "I thought we had the convention that if the the tag 'CostCenter' == 'Other' then there must be a second tag 'CostCenterOther' with the actual cost center.  Why doesn't this PR follow the convention?"

Reply:
@brucehoff We only used that convention when provisioning with the service catalog due to the 25 tag option limitation.  This is the PR provisioner system which does not have that limitation so we don't need to use that convention here.

Reply:
@zaro0508 I thought the only reason we use "Other / 000001" is to indicate that the actual cost center is in the "CostCenterOther" tag.  If so, we shouldn't use "Other / 000001" here, we should use the actual program code or "NO PROGRAM / 00000".  If I am misunderstanding, let me know.